page_example/page_example.module

```diff
@@ -72,15 +72,29 @@ function page_example_permission() {
 function page_example_menu() {
 
   // This is the minimum information you can provide for a menu item. This menu
-  // item will be created in the default menu, usually Navigation.
-  $items['examples/page_example'] = array(
-    'title' => 'Page Example',
+  // item will be created in the default internal menu, which is invisible in the 
+  // user interface.
+  $items['examples/page_menu_internal'] = array(
+    'title' => 'Page Menu Internal',
     'page callback' => 'page_example_description',
     'access callback' => TRUE,
     'expanded' => TRUE,
   );
 
-  $items['examples/page_example/simple'] = array(
+  // You can add a menu item to a specific menu. In this case, we will add it
+  // to the primary navigation (main-menu).
+
+  $items['examples/page_menu_visible'] = array(
+    'title' => 'Page Menu Visible',
```

Comment:
*Page Example: Visible Menu* would be a better title which makes clear from where the menu item comes from (as most of the strings used by the examples modules.)